l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -26,6 +29,18 @@ def id_generator(
     return "".join(random.choice(chars) for dummy in range(size))
 
 
+def format_numbers(number):
```

Comment:
Questa funzione è duplicata di https://github.com/TheMule71/l10n-italy/blob/fec1bc1bc4a832127bd63596480bf62a1579ae9e/l10n_it_fatturapa_out/wizard/efattura.py#L79
Per evitare duplicazione di codice non sarebbe meglio spostare tutte le [funzioni](https://github.com/TheMule71/l10n-italy/blob/fec1bc1bc4a832127bd63596480bf62a1579ae9e/l10n_it_fatturapa_out/wizard/efattura.py#L69-L219) in un file dedicato (es. utils.py o efattura_utils.py) e importare  `format_numbers` in wizard_export_fatturapa.py?

Reply:
Beh, efattura.py è un file che dovrebbe servire a quello, le funzioni sono definite locali ma potrebbero essere di modulo. Per adesso mi limito a spostare `format_numbers` per risolvere il problema specifico (nell'altro file la importo).

Reply:
:+1: